CONTRIBUTING.md

```diff
@@ -4,38 +4,77 @@ New Relic welcomes code contributions by the Node community to this module, and
 have taken effort to make this process easy for both contributors and our
 development team.
 
+## Processes
+
+## Feature Requests
+
+Feature requests should be submitted in the [Issue tracker](../../issues), with
+a description of the expected behavior & use case. Before submitting an Issue,
+please search for similar ones in the [closed
+issues](../../issues?q=is%3Aissue+is%3Aclosed+label%3Aenhancement).
+
+### Code of Conduct
+
+Before contributing please read the [code of conduct](./CODE_OF_CONDUCT.md)
+
+Note that our [code of conduct](./CODE_OF_CONDUCT.md) applies to all platforms
+and venues related to this project; please follow it in all your interactions
+with the project and its participants.
+
+### Contributor License Agreement
+
+Keep in mind that when you submit your Pull Request, you'll need to sign the
+CLA via the click-through using CLA-Assistant. If you'd like to execute our
+corporate CLA, or if you have any questions, please drop us an email at
+opensource@newrelic.com.
+
+For more information about CLAs, please check out Alex Russell’s excellent
+post, [“Why Do I Need to Sign
+This?”](https://infrequently.org/2008/06/why-do-i-need-to-sign-this/).
+
+### Slack
+
+We host a public Slack with a dedicated channel for contributors and
+maintainers of open source projects hosted by New Relic.  If you are
+contributing to this project, you're welcome to request access to the
+#oss-contributors channel in the newrelicusers.slack.com workspace.  To request
+access, see https://newrelicusers-signup.herokuapp.com/.
+
+## The Code
+
+### Version Support
+
 When contributing, keep in mind that New Relic customers (that's you!) are
 running many different versions of Node, some of them pretty old. Changes that
 depend on the newest version of Node will probably be rejected, with prejudice
 if they replace something backwards compatible.
 
-Also be aware that the instrumentation needs to work with a wide range of
-versions of the instrumented modules, and that code that looks nonsensical or
+Be aware that the instrumentation needs to work with a wide range of versions
+of the instrumented modules, and that code that looks nonsensical or
 overcomplicated may be that way for compatibility-related reasons. Read all the
 comments and check the related tests before deciding whether existing code is
 incorrect.
 
-## Testing
+### Testing
 
 The agent includes a suite of unit and functional tests which should be used to
 verify your changes don't break existing functionality.
 
 Unit tests are stored in `test/`. They're written in
-[Mocha](https://mochajs.org), and have the extension
-`.test.js`.
+[Mocha](https://mochajs.org), and have the extension `.test.js`.
```

Comment:
Technically, we are using `tap` now with some left behind using the mocha DSL until we finish converting.

Reply:
Pushed an update with this removed